react/states_and_effects/introduction_to_state.md

```diff
@@ -20,6 +20,14 @@ Let's take a look at how we can use state in our code.
 
 <iframe style="border: 1px solid rgba(0, 0, 0, 0.1);border-radius:2px;" width="100%" height="450" src="https://codesandbox.io/p/sandbox/unruffled-pasteur-6h36wf?file=%2Fsrc%2FApp.jsx%3A10%2C27&embed=1" allowfullscreen></iframe>
 
+<div class="lesson-note lesson-note--critical" markdown="1">
+
+If you are unable to view the above CodeSandbox, fork it to your own CodeSandbox account by clicking the "fork" button.
```

Comment:
```suggestion
If you are unable to view the above CodeSandbox, fork it to your own CodeSandbox account by clicking the "fork" button.

If you still get a "Failed to fork: (401)" error, click the "fork" button again.

```
Quite a lot of people have run into a 401 after the first fork, but the second fork's worked a charm every time that's happened.